types/win-ca/win-ca-tests.ts

```diff
@@ -0,0 +1,11 @@
+import ca, { der2 } from "win-ca";
```

Comment:
this is CJS module, so that won't work here on DT, only when consuming and using in TS projects.
here one needs:
```
import ca = require('win-ca');
```
(as per 'win-ca' readme btw) and `ca.der2` or import `der2` directly or in separate import line (adding lint eclusion for duplicate imports in tests files if required)